web/themes/custom/recipe_theme/src/sass/layout/_page.footer.scss

```diff
@@ -1,6 +1,6 @@
 // Page: Footer
 // -----------------------------------------------------------------------------
 .page__footer {
-  $body-bg: $gray-100 !default;
-  $body-color: $gray-900 !default;
+  $body-bg: #273238;
```

Comment:
Please use variables for colors based on what you have in figma, like `$brand-1`, `$brand-2` etc.

Reply:
I used the color codes from Figma, but it's more difficult to memorize a "brand-1" named variable, I can't identify them later,  that's why I used easier names, but the colors behind the variables are the brand colors. Do I have to change them for sure?

Reply:
That depends. Are hexa codes easier to remember? :)

![Screenshot_20200430_130837](https://user-images.githubusercontent.com/13573052/80703788-b9f0d980-8ae3-11ea-9e36-49088b071413.png)

As I can see in figma these brands also have a numbered identifier attached to them. The reason why variables are preferred because once you have to change one of these colors, if you use variables you change the value of the variable. If you use hexa colors, you need to change each and every hexa color used in the code.

When you refer to variable names, you don't need to know its value, only what it's used for. At Pronovix we only use variables for colors.

Lastly, if some colors come from variables, some colors use the exact hex value then the code will become inconsistent and harder to read. If variables are used everywhere, the code will be more consistent.

Reply:
There is a fun fact with Bootstrap, its variables file already contains predefined color values which are used for numerous bootstrap components on the site:

* [Gray values](https://github.com/Pronovix/food/blob/master/web/themes/custom/recipe_theme/src/sass/base/_variables.scss#L9)
* [Brand values](https://github.com/Pronovix/food/blob/master/web/themes/custom/recipe_theme/src/sass/base/_variables.scss#L69)

You can also modify the values here with the ones in Figma, and then reuse thes variables, e.g. `$primary` and `$secondary`.